### PR TITLE
add en_SG documentation

### PR DESCRIPTION
**Pull request by IonBazan**
This PR adds missing documentation for English (Singapore) - `en_SG` providers.

### CODE REVIEW

**Comment by pimjansen** [comment]
A lot of these things are already in core docs isnt it? Try only to document the locale slecific

**Comment by IonBazan** [comment]
@pimjansen Not really:

- `floorNumber`, `apartmentNumber`, `streetNumber`, `blockNumber` and `streetPrefix` are only available in this locale
- `nric`, `fin` and `singaporeId` are Singapore-specific
- `internationalCodePrefix`, `voipNumber`, `premiumPhoneNumber` are added by this locale
-  `tollFreeInternationalNumber` and `tollFreeLineNumber` are different from `en_US`'s `tollFreeNumber`
- `mobileNumber` and `fixedLineNumber` are different from `en_US`'s `phoneNumber` which is choosing one of them in `en_SG` case.

**Comment by pimjansen** [comment]
EnUS is also not core, see the root providers

**Comment by IonBazan** [comment]
@pimjansen The documentation is always using `en_US` for these [examples](https://github.com/FakerPHP/fakerphp.github.io/blob/main/docs/formatters.md). 
There is currently no documentation for root `Address`, `PhoneNumber` and `Person` providers in [docs/formatters ](https://github.com/FakerPHP/fakerphp.github.io/blob/main/docs/formatters) directory.

**Comment by krsriq** [comment]
Couldn't find any providers that have public `*Prefix` (or `*Suffix`) methods that are documented (except for a couple in en_US and in ro_RO/Person), even though there are a lot that use prefixes/suffixes.

If we document `streetPrefix` here we should probably document _all_ `*Prefix`/`*Suffix` methods (that are not in the root providers), which would make things a bit noisy.

**Comment by IonBazan** [comment]
@krsriq Thank you for your suggestion. I've removed non-important methods documentation and left only `blockNumber` for address.

File: docs/locales/en_SG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+```php
+echo $faker->blockNumber; // "Blk 914"
+```
+
+### `Faker\Provider\en_SG\Person`
+
+```php
+// Generates a National Registration Identity Card (NRIC) number
+echo $faker->nric; // "S4098150J"
```
Comment (bramceulemans) [inline docs/locales/en_SG.md:13]: Please use single quotes.

**Review by bramceulemans — CHANGES_REQUESTED** [review]
Left you some comments, otherwise this looks fine.

**Review by bramceulemans — APPROVED** [review]

**Pull request merged**